scrypto/src/test/scala/sparkz/crypto/signing/SigningFunctionsSpecification.scala

```diff
@@ -15,45 +15,23 @@ class SigningFunctionsSpecification extends AnyPropSpec
     forAll { (seed1: Array[Byte], seed2: Array[Byte],
               message1: Array[Byte], message2: Array[Byte]) =>
       whenever(!seed1.sameElements(seed2) && !message1.sameElements(message2)) {
-        val keyPair = Curve25519.createKeyPair(seed1)
-        val keyPair2 = Curve25519.createKeyPair(seed2)
+        val keyPair = Ed25519.createKeyPair(seed1)
+        val keyPair2 = Ed25519.createKeyPair(seed2)
 
-        val sig = Curve25519.sign(keyPair._1, message1)
+        val sig = Ed25519.sign(keyPair._1, message1)
 
-        Curve25519.verify(sig, message1, keyPair._2) shouldBe true
-        Curve25519.verify(sig, message1, keyPair2._2) should not be true
-        Curve25519.verify(sig, message2, keyPair._2) should not be true
+        Ed25519.verify(sig, message1, keyPair._2) shouldBe true
+        Ed25519.verify(sig, message1, keyPair2._2) should not be true
+        Ed25519.verify(sig, message2, keyPair._2) should not be true
 
       }
     }
   }
 
-  property("shared secret should be same for both parties ") {
```

Comment:
Why doesn't the new `Ed25519` class have a `createSharedSecret` method? Is it something we want to drop?

Reply:
The old method Curve25519.createSharedSecret was not used anywhere, so I dropped it.